src/js/core/grid.js

```diff
@@ -106,16 +106,17 @@
 
     // helper
 
-    UI.Utils.matchHeights = function(elements, options) {
+    UI.Utils.matchHeights = function(elements, options, parent) {
 
         elements = $(elements).css('min-height', '');
         options  = $.extend({ row : true }, options);
 
         var firstvisible = elements.filter(":visible:first");
+        parent = parent || firstvisible.parent();
 
         if (!firstvisible.length) return;
 
-        var stacked      = Math.ceil(100 * parseFloat(firstvisible.css('width')) / parseFloat(firstvisible.parent().css('width'))) >= 100 ? true : false,
+        var stacked      = Math.ceil(100 * parseFloat(firstvisible.css('width')) / parseFloat(parent.css('width'))) >= 100 ? true : false,
```

Comment:
Couldn't this just be
`var stacked = parseFloat(firstvisible.css('width')) <= parseFloat(parent.css('width')),`?